mandatory/2-function-creation.js

```diff
@@ -15,26 +22,53 @@ function isAcceptableUser(userAge, isLoggedIn) {}
     is applieds and 142.5 should be returned)
 */
 
-function applyDiscount(totalPrice) {}
+  
+function applyDiscount(totalPrice) {
+  if (totalPrice > 200) {
+    return totalPrice * 0.9;
+  } else {
+    return totalPrice * 0.95; 
+  }
+}
 
 /*
   Complete the function to print to the console the odd numbers between 1 and limit (use a while loop):
   */
-function printOddNumbers(limit) {}
+function printOddNumbers(){
+  let limit = 1
+  while (limit <= 10){
+    console.log(limit)
+    limit+=2
+  }
+}
 
 /*
   Complete the buyTwoGetTheCheapestFree function: if user buys two items, the cheapest item will be free!
   The function should return the price to be paid once the discount is applied.
 */
-function buyTwoGetTheCheapestFree(price1, price2) {}
-
+function buyTwoGetTheCheapestFree(price1, price2) {
+    if (price1 > price2){
+      return price2 = price1;
```

Comment:
Have a think about what is happening on this line of code - you're doing: `return price2 = price1;` - which means you are assigning the value of `price1` to the `price2` variable, before returning.
But do you need to do this assigning? Can you just return one of the values directly?